src/firefly/java/edu/caltech/ipac/firefly/server/db/DuckDbUDF.java

```diff
@@ -17,12 +17,12 @@
 public class DuckDbUDF {
     public static String deg2pix = importDeg2Pix();
     public static final String decimate_key = """
-        CREATE FUNCTION decimate_key(xVal, yVal, xMin, yMin, nX, nY, xUnit, yUnit) AS
+        CREATE OR REPLACE FUNCTION decimate_key(xVal, yVal, xMin, yMin, nX, nY, xUnit, yUnit) AS
```

Comment:
Is this related to the stability issue?

Reply:
No.  This occurs only in edge cases. I included it to ensure the create function never fails and always contains the latest version.

Reply:
The latest version of DuckDB appears to fix some of the unexpected crashing issue.